lib/served/resource/requestable.rb

```diff
@@ -39,6 +39,10 @@ def initialize
           '{/resource*}{/id}.json{?query*}'
         end
 
+        class_configurable :raise_on_exceptions do
+          true
+        end
+
```

Comment:
by this, we should not raise on exceptions from the resource unless `save!` is called, unlike how we currently do it where `save` rescues